aws/ssm/parameters.go

```diff
@@ -0,0 +1,41 @@
+package ssm
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	log "github.com/sirupsen/logrus"
+)
+
+// GetParameter takes in the variable key and checks OS for it
+// if there is no environment variable for the key, it will pull
+// from Parameter Store (Encrypted)
+// Variables in parameter store must all be encrypted
+func GetParameter(key string) (string, error) {
+	value := os.Getenv(key)
+	if value != "" {
+		return value, nil
+	}
+	withDecryption := true
+	sess, err := session.NewSessionWithOptions(session.Options{
+		Config:            aws.Config{Region: aws.String("us-west-2")},
```

Comment:
us-west-2 should go to some env variable IMHO

Reply:
done.